core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -434,6 +434,12 @@ public static ProcessTree get() {
                     vetoersExist = channelToMaster.call(new DoVetoersExist());
                 }
             }
+            catch (InterruptedException ie) {
```

Comment:
You should at least set the thread's interrupt flag back so that code further down the line won't try to block again without being immediately re-interrupted.

Reply:
Good point.

Reply:
It makes the most sense to be explicit about clearing the interrupt flag and letting the flow continue on.

If this operation is interrupted, that means we can't determine if vetoers exist. The most likely scenario is that the condition will continue and waiting or anything else won't enable us to determine if vetoers exist. Continuing with no vetoers is the best we can do.